http-server-jetty/src/main/java/io/micronaut/servlet/jetty/JettyFactory.java

```diff
@@ -246,7 +246,7 @@ public Resource addPath(String path) throws IOException {
             resourceHandler.setCacheControl(config.getCacheControl());
         }
 
-        ContextHandler contextHandler = new ContextHandler(path + "/*");
+        ContextHandler contextHandler = new ContextHandler(path);
         contextHandler.setContextPath("/");
```

Comment:
Note: potentially we could remove the duplicate path fully, because after the constructor call the `contextPath` is immediately overwritten:

```suggestion
        ContextHandler contextHandler = new ContextHandler();
        contextHandler.setContextPath("/");
```
or
```suggestion
        ContextHandler contextHandler = new ContextHandler("/");
```

however I'm not sure how the `config.getMapping()` will get used then.

Reply:
Potentially related? https://github.com/micronaut-projects/micronaut-servlet/issues/114

Reply:
Good shout, but #114 works with Micronaut 4, so I closed it 👍